ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -1229,39 +1208,12 @@ MmCreateProcessAddressSpace(IN ULONG MinWs,
     Process->AddressSpaceInitialized = 1;
 
     /* Set the base directory pointers */
-    Process->WorkingSetPage = WsListIndex;
     DirectoryTableBase[0] = PdeIndex << PAGE_SHIFT;
     DirectoryTableBase[1] = HyperIndex << PAGE_SHIFT;
 
     /* Make sure we don't already have a page directory setup */
     ASSERT(Process->Pcb.DirectoryTableBase[0] == 0);
 
-    /* Get a PTE to map hyperspace */
```

Comment:
Isn't that part of the code covered by the other PR 3572?

Reply:
Yes. One of the two will eventually need to be rebased. But this is needed to have both branches in a working state.